### PR TITLE
Adds space wasps

### PR DESCRIPTION
**Pull request by Tomeno**
Icon by MrSnapwalk

### CODE REVIEW

**Comment by ghost** [comment]
I like it, good job Snap.

File: code/modules/mob/living/simple_animal/hostile/wasp.dm [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	maxHealth = 30
+	health = 30
+
+	harm_intent_damage = 8
+	melee_damage_lower = 8
+	melee_damage_upper = 12
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+
+	//Space carp aren't affected by atmos.
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/wasp.dm]: _cough_ remove this _cough_

Copypasting Simple animals is generally fine, due to them being fine code already, but atleast update the comments/remove them.

**Comment by Tomeno** [comment]
Do not merge yet - I will add stuff.

**Comment by Atskadan** [comment]
Shit, I literally read that right as I pressed the button.
Sorry. You can easily just make a new PR for the new stuff.

**Pull request merged**